### ISSUE TITLE
--noobaa-image cli flag only works when before commands

### ISSUE THREAD

**Issue by guymguym** [noobaa/noobaa-operator#18]
this works:
```
./noobaa --noobaa-image=bla system yaml
```

but this doesn't update the image in the CR:
```
./noobaa  system yaml --noobaa-image=bla
```

Maybe related to using a `cobra.Command.PersistentFlags()` incorrectly?

**Comment by guymguym** [comment]
To clarify, this issue happened to @dannyzaken when trying to use a local image:
```
nb install --noobaa-image=danny:danny
```
and it installed the CR with the default image (noobaa/noobaa-operator:5).

And when we moved the flag to before the commands it worked:
```
nb --noobaa-image=danny:danny install
```

Need investigation.

**Comment by guymguym** [comment]
Fixed by #24
